static/app/components/githubFeedbackButton.tsx

```diff
@@ -0,0 +1,16 @@
+import {LinkButton, LinkButtonProps} from 'sentry/components/button';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconGithub} from 'sentry/icons';
+import {t} from 'sentry/locale';
+
+type GithubFeedbackButtonProps = LinkButtonProps & {href: string};
+
+const title = t('Give us feedback on GitHub');
+
+export function GithubFeedbackButton(props: GithubFeedbackButtonProps) {
+  return (
+    <Tooltip title={title}>
+      <LinkButton size="sm" external icon={<IconGithub />} {...props} />
```

Comment:
Should the text of the button be optional and default to `t('Give Feedback')`?

Reply:
hmm, then icon-only usage is not possible anymore but maybe we never want this 🤔

Reply:
That's a good point. Would you be able to pass null for children if you want just an icon button?

Reply:
You could but then the buttons typescript types will not force you to add an `aria-label`.